stable/aws-efa-k8s-device-plugin/templates/daemonset.yaml

```diff
@@ -67,7 +67,12 @@ spec:
           volumeMounts:
             - name: device-plugin
               mountPath: /var/lib/kubelet/device-plugins
+            - name: infiniband-volume
```

Comment:
> 0.5.1. As part of this new version, the EFA plugin needs to be mounted to /dev/infiniband; 

Could you share the reference to this?

Reply:
From the README in https://github.com/linux-rdma/rdma-core, we need to explicitly mount `/dev/infiniband` since that is where rdma-core accesses device nodes.